reaktive/src/commonMain/kotlin/com/badoo/reaktive/maybe/DoOnAfter.kt

```diff
@@ -73,17 +73,17 @@ fun <T> Maybe<T>.doOnAfterSuccess(action: (T) -> Unit): Maybe<T> =
                     emitter.onSuccess(value)
 
                     // Can't send error to downstream, already terminated with onComplete
-                    tryCatchAndHandle { action(value) }
+                    tryCatchAndHandle { consumer(value) }
                 }
             }
         )
     }
 
 /**
- * Calls the [action] when the [Maybe] signals `onComplete`.
- * The [action] is called **after** the observer is called.
+ * Calls the [consumer] when the [Maybe] signals `onComplete`.
+ * The [consumer] is called **after** the observer is called.
  */
-fun <T> Maybe<T>.doOnAfterComplete(action: () -> Unit): Maybe<T> =
+fun <T> Maybe<T>.doOnAfterComplete(consumer: () -> Unit): Maybe<T> =
```

Comment:
It is not a consumer because it does not consume anything, so the name `action` is correct in this case. Could you please revert this change?

Reply:
Done